src/controls/sort.tsx

```diff
@@ -168,6 +169,14 @@ function getSortFunctionParameterized(
               : -1;
         }
       };
+
+    case SortBy.Date:
+      return (a, b) => {
+        const dateA = getDate(a);
+        const dateB = getDate(b);
+
+        return dateB - dateA; // Sorts descending (newest first)
+      };
```

Comment:
You can compress the code and its just as readable.

```suggestion
      return (a, b) => getDate(b) - getDate(a);
```